src/PositionManager.sol

```diff
@@ -161,6 +164,14 @@ contract PositionManager is
         }
     }
 
+    /// @dev integrators may elect to forfeit positive deltas with clear
+    /// provide a safety check that amount-to-clear is less than a user-provided maximum
+    function _clear(Currency currency, uint256 amountMax) internal {
+        int256 currencyDelta = poolManager.currencyDelta(address(this), currency);
+        if (amountMax < uint256(currencyDelta)) revert ClearExceedsMaxAmount(currency, currencyDelta, amountMax);
```

Comment:
this is preference but it reads easier to me if we flip the vars

if (currencyDelta > amountMax) revert..